docs/internal/Assignment-Process.md

```diff
@@ -75,81 +72,54 @@ Available for either a Reviewer that can do **self-assignment** or an Assigner w
-
+- For previous assignment set `review_assignment.trigger` to ON_REVIEW_UNASSIGN
+- Set the status of previous assignment `review_assignment` to **Available**.
+- And `assigned_sections` are set to empty array in the back-end.
+2. Else for assignment with sections still assigned to previous assignment
+- Set `review_assignment.trigger` to ON_REVIEW_ASSIGN
 - Just update the previous assignment `assigned_sections` but keep status as **Assigned**.
 
-3. Updates the new assignment
-
-- Set the `review_assignment.trigger` to ON_REVIEW_REASSIGN.
-- Changes status of new `review_assignment` to **Assigned**.
-- Sets assignerId of current user in the new assignment.
-- Sets `assigned_sections` on record - it means that all elements of type QUESTION in these sections are also assigned
-
-7. Other updates done on the server-side:
-   - If an existing `review` was connected to previous assignment UNASSIGNED it will change the `review.status` from **Draft** to **Discontinued**.
+#### Other updates done on the server-side after re-assignment
+   - If an existing `review` was connected to previous assignment now UNASSIGNED it will change the `review.status` from **Draft** to **Discontinued**.
    - If an existing `review` was connected to new assignment REASSIGNED it will change the `review.status` from **Discontinued** to **Draft**.
+### Unassignment
+The same GraphQL mutation `updateReviewAssignment` is called in the front-end after the assigner user clicks on `unassign` options (in one of the sections) and confirms the warning modal clicking on the **Confirm** button.
 
-**Note**: No need to change other assignment flag `is_locked` since it has been assigned to reviewers already.
-
-#### Unassignment
-
-A GraphQL mutation `unassignReviewAssignment` runs on the front-end when one assigner:
-
-- selects **un-assign**
-- confirms unassignment of reviewer for each section (or all sections when rules apply).
+- Assigner/Current reviewer clicks on **unassign** option in one of the sections
+- A warning modal is presented to confirnm unassignment (from all sections this user is assigned)
+- User clicks on the **Confirm** button to register the change
 
+#### After the Assigner or Reviewer click on Confirm after unassigning
 1. `review_assignment.trigger` is set to ON_REVIEW_UNASSIGN.
-2. Changes status of `review_assignment` to **Available** and flag `is_locked` to **False**.
-3. Set `assigned_sections` to empty array
-4. Other updates done on the server-side:
+2. Changes status of `review_assignment` to **Available**
+3. Set `assigned_sections` to empty array (done in back-end)
+
+#### Other updates done on the server-side after unassignment
    - If an existing `review` was connected to this assignment will change the `review.status` from **Draft** to **Discontinued**.
-   - Update other `review_assignment` with `is_self_assignment` as **True** to have field `is_locked` to **False**. So other self-assignments for same application are unlocked.
 
```

Comment:
Ok - Something is not displaying the formatting well here. Better to look at the changes on your IDE (using a tool to check the .md files formatted. Here is the example of above changes. Simplifying the overview on Front-end - and changes done in back-end - for processes of Self-assigning/Assigning, Re-assigning and Un-assignment.

![Screen Shot 2022-11-17 at 11 37 41 AM](https://user-images.githubusercontent.com/16461988/202309937-dceb5895-369b-46ca-b6b6-d8aea0240d40.png)

![Screen Shot 2022-11-17 at 11 38 01 AM](https://user-images.githubusercontent.com/16461988/202309935-9c1be43e-ecab-4f03-b395-b0c9405fa4ad.png)

![Screen Shot 2022-11-17 at 11 38 07 AM](https://user-images.githubusercontent.com/16461988/202309798-3bfc3892-3410-455f-a526-cde0dce651e2.png)